### PR TITLE
deprecating the image py-ews

### PR DESCRIPTION
**Pull request by maimorag**
## Status
In Progress

## Related Content Pull Request
Related PR: [link to the PR at demisto/content](https://github.com/demisto/content/pull/33261)

## Related Issues
Related: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-9764)

## Description
to reuse py3ews, I add the necessary packs from py-ews to py3ews.
Also, I added a fix in ewsv2 PR.

### CODE REVIEW

File: docker/py3ews/Pipfile [hunk 1]
```diff
@@ -6,9 +6,14 @@ verify_ssl = true
 [dev-packages]
 
 [packages]
-exchangelib = "*"
+exchangelib = "<=5.0.3"
```
Comment (JudahSchwartz) [inline docker/py3ews/Pipfile:9]: why are we locking this?

**Review by JudahSchwartz — COMMENTED** [review]
did you check that the other integrations using this image work with the upgraded exchangelib?

File: docker/py3ews/verify.py [hunk 2]
```diff
@@ -37,4 +37,4 @@
 assert 'md4' in hashlib.algorithms_available
 hashlib.new('md4', b"text")
 
-print('all is good, `get_localzone() -> {}` is working'.format(test))
+print('all good, `get_localzone() -> {}` is working'.format(test))
```
Comment (eyalpalo) [inline docker/py3ews/verify.py:40]: Why do we change it?

**Review by eyalpalo — APPROVED** [review]
Looks good, please ask @JudahSchwartz to approve also before merging.

File: docker/py3ews/verify.py (hunk 2 above)
Comment (JudahSchwartz) [reply docker/py3ews/verify.py:40]: to satisfy the "update the verify file check" 😂 
please add imports for the relevant packages instead

**Review by JudahSchwartz — APPROVED** [review]
looks good, please note my comment

**Review by JudahSchwartz — CHANGES_REQUESTED** [review]
please note the validation about the native image

**Review by eyalpalo — CHANGES_REQUESTED** [review]
Please address @JudahSchwartz comments, add the libraries to the verify file and make sure all validations pass

**Review by JudahSchwartz — APPROVED** [review]
we will need to open the pr on content side to ignore ewsv2 from native image version that is out right now. We can add label after that is done

**Review by eyalpalo — APPROVED** [review]

**Pull request merged**